From 85c172260805e69a8f9997bcf4f24a6af4e60d6c Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 10 May 2016 11:18:45 +0800
Subject: [PATCH] ARM64: rockchip_cros_defconfig: enable the regulator LP8752

The gpu can't work with chromeos since the regulator has not enabled.
Okay, the gpu can work for chromeos with rk3399 evb2 on now.

Change-Id: If23affff151a99206b86c2a3f40c21752bdc2d54
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 75604e8c24c7..096c5f4a3f5b 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -339,6 +339,7 @@ CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_ACT8865=y
 CONFIG_REGULATOR_FAN53555=y
 CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_LP8752=y
 CONFIG_REGULATOR_MP8865=y
 CONFIG_REGULATOR_PWM=y
 CONFIG_REGULATOR_RK808=y
-- 
2.35.3

